iroha/src/sumeragi.rs

```diff
@@ -149,16 +155,17 @@ impl Sumeragi {
             self.proxy_tail().clone(),
         )
         .await;
-        Ok(())
+        Ok(block)
     }
 
     pub async fn handle_message(&mut self, message: Message) -> Result<(), String> {
```

Comment:
Do we assume that the block has been fully validated before we handle this?

Reply:
I don't see why we should return the block. From what I understand, when the block is created we only send the `Created` message and then wait for `Commited` message to actually commit the block to Kura and WSV.

Reply:
Right now - we do not use this method at all. So I think we will make changes after adoption